control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -1409,6 +1488,155 @@ func (r *HostedControlPlaneReconciler) reconcilePKI(ctx context.Context, hcp *hy
 	return nil
 }
 
+func (r *HostedControlPlaneReconciler) reconcileKubevirtCSIDriver(ctx context.Context, hcp *hyperv1.HostedControlPlane, createOrUpdate upsert.CreateOrUpdateFN) error {
+	// var crdResource = schema.GroupVersionResource{Group: "apiextensions.k8s.io", Version: "v1", Resource: "customresourcedefinitions"}
+	// r.TenantDynamicClient.Resource(crdResource).Create(ctx, kubevirt_assets.CsiDriverCRD.(*unstructured.Unstructured), metav1.CreateOptions{})
+
+	// apixClient, err := apixv1beta1client.NewForConfig(r.TenantRestConfig)
+	// if err != nil {
+	// 	return fmt.Errorf("failed to build rest config: %w", err)
+	// }
+	// crds := apixClient.CustomResourceDefinitions()
+	// crds.Cr
+	infraNamespace := hcp.Namespace
+	// infraUrl := r.RestConfig.APIPath
+	// infraCertificateAuthorityData := string(r.RestConfig.CAData)
+	// infraToken := "TODO"
+	infraKubeconfigSecretName := "tenant-controller-kubeconfig"
```

Comment:
this secret needs a better name that signifies that it is related to the kubevirt csi driver